File: yeongseong/4week/로그인 성공.js
function solution(id_pw, db) {
    for (let i = 0; i < db.length; i++) {
      if (id_pw[0] === db[i][0]) {  // 입력된 아이디와 회원의 아이디 비교
        if (id_pw[1] === db[i][1]) {  // 입력된 비밀번호와 회원의 비밀번호 비교
          return "login";  // 아디디와 비밀번호가 모두 일치하는 경우 login
        } else {
          return "wrong pw";  // 아이디는 일치하지만 비밀번호가 일치하지 않는 경우 wrong pw
        }
      }
    }
    return "fail";  // 일치하는 아이디가 없는 경우 fail
  }
  

File: gang/week4/로그인 성공.java
class Solution {
    public String solution(String[] id_pw, String[][] db) {
        String answer = "";
        for(int i=0; i< db.length; i++) {       // id가 같은지 확인
            if(id_pw[0].equals(db[i][0])) {     // id가 같은데 pw도 같으면 login
                if(id_pw[1].equals(db[i][1])) {
                    answer = "login";
                    break;
                }
                else {                          // id는 같지만 pw가 다르면 wrong pw
                    answer = "wrong pw";
                    break;
                }
            }
            else {                              // id가 다르면 fail
                answer = "fail";
            }
        }
        return answer;
    }
}